src/Constants.h

```diff
@@ -45,4 +45,12 @@ const double GPS_WAYPOINT_RADIUS = Constants::ROBOT_LENGTH * 1.5;
 const double LANDMARK_WAYPOINT_RADIUS = Constants::ROBOT_LENGTH * 1.3;
 const double EPS = 2.0; // heuristic weight for weighted A*
 }
+
+// Lidar
+namespace Lidar {
+const std::string RP_PATH = "/dev/ttyUSB0";
+constexpr double MM_PER_M = 1000;
+constexpr uint32_t RPLIDAR_A1_BAUDRATE = 115200;
+constexpr uint32_t RPLIDAR_S1_BAUDRATE = 115200;
```

Comment:
We are using 256000 baud for the S1 (since I think the serial adapter is broken)
```suggestion
constexpr uint32_t RPLIDAR_S1_BAUDRATE = 256000;
```